diff -aurN totem-3.8.0.orig/configure.ac totem-3.8.0/configure.ac
--- totem-3.8.0.orig/configure.ac	2013-03-28 23:53:17.415503643 +0100
+++ totem-3.8.0/configure.ac	2013-03-28 23:53:59.257977565 +0100
@@ -144,7 +144,7 @@
 	AC_MSG_RESULT([yes])
 else
 	AC_MSG_RESULT([no])
-	AC_MSG_ERROR([
+	AC_MSG_WARN([
 		Cannot find required GStreamer-1.0 tool 'gst-inspect-1.0'.
 		It should be part of gstreamer-1_0-utils. Please install it.
 	])
@@ -160,7 +160,7 @@
 		AC_MSG_RESULT([yes])
 	else
 		AC_MSG_RESULT([no])
-		AC_MSG_ERROR([
+		AC_MSG_WARN([
 			Cannot find required GStreamer-1.0 plugin '$1'.
 			It should be part of gst-plugins-base. Please install it.
 		])
@@ -179,7 +179,7 @@
 		AC_MSG_RESULT([yes])
 	else
 		AC_MSG_RESULT([no])
-		AC_MSG_ERROR([
+		AC_MSG_WARN([
 			Cannot find required GStreamer-1.0 plugin '$1'.
 			It should be part of gst-plugins-good. Please install it.
 		])
